### PR TITLE
removed public subnets

### PR DESCRIPTION
**Pull request by AsysGuptaSF**
Removed public subnets and NAT gateways from production VPC.

### CODE REVIEW

File: hipaa/network-config.yaml [hunk 1]
```diff
@@ -459,18 +459,18 @@ vpcs:
...
-            destination: 0.0.0.0/0
-            type: internetGateway
-            target: IGW
-      - name: HIS-WL-Prod-Public-App-B
-        routes:
-          - name: IgwRoute
-            destination: 0.0.0.0/0
-            type: internetGateway
-            target: IGW
+      # - name: HIS-WL-Prod-Public-App-A
```
Comment (jamescrowley321) [inline hipaa/network-config.yaml:462]: Remove commented out code.

File: hipaa/network-config.yaml [hunk 2]
```diff
@@ -480,14 +480,14 @@ vpcs:
...
         ipv4CidrBlock: 10.3.1.0/24
-      - name: HIS-WL-Prod-Public-App-A
-        availabilityZone: a
-        routeTable: HIS-WL-Prod-Public-App-A
-        ipv4CidrBlock: 10.3.2.0/24
-      - name: HIS-WL-Prod-Public-App-B
-        availabilityZone: b
-        routeTable: HIS-WL-Prod-Public-App-B
-        ipv4CidrBlock: 10.3.3.0/24
+      # - name: HIS-WL-Prod-Public-App-A
```
Comment (jamescrowley321) [inline hipaa/network-config.yaml:483]: Remove commented out code.

File: hipaa/network-config.yaml [hunk 3]
```diff
@@ -496,11 +496,11 @@ vpcs:
         availabilityZone: b
         routeTable: HIS-WL-Prod-Tgw-B
         ipv4CidrBlock: 10.3.255.224/28
-    natGateways:
-      - name: Nat-HIS-WL-Prod-Public-App-A
-        subnet: HIS-WL-Prod-Public-App-A
-      - name: Nat-HIS-WL-Prod-Public-App-B
-        subnet: HIS-WL-Prod-Public-App-B
+    # natGateways:
```
Comment (jamescrowley321) [inline hipaa/network-config.yaml:499]: Remove commented out code.

**Review by jamescrowley321 — CHANGES_REQUESTED** [review]

File: hipaa/network-config.yaml (hunk 1 above)
Comment (AsysGuptaSF) [reply hipaa/network-config.yaml]: Done.

File: hipaa/network-config.yaml (hunk 2 above)
Comment (AsysGuptaSF) [reply hipaa/network-config.yaml]: Done.

File: hipaa/network-config.yaml (hunk 3 above)
Comment (AsysGuptaSF) [reply hipaa/network-config.yaml]: Removed commented out code.

**Pull request merged**